scripts/check_code_committed.js

```diff
@@ -0,0 +1,28 @@
+const { execSync } = require('child_process');
+const { simpleGit } = require('simple-git');
+
+const [,,command] = process.argv;
+
+if (!command) {
+  throw new Error('Missing argument for script, please provide an action to perform.')
+}
+
+execSync(command, {stdio: 'inherit'});
+
+(async () => {
+  const status = await simpleGit().status();
+  console.log('git status', status);
+  if (!isEmpty(status)) {
+    throw new Error(`There are uncommited file changes for script "${command}":\n${JSON.stringify(status, null, 2)}`);
```

Comment:
I don't think we need this JS file. Based on the answers here 
https://stackoverflow.com/questions/5139290/how-to-check-if-theres-nothing-to-be-committed-in-the-current-branch
we could run
```
git diff --exit-code
```
and let the pipeline fail in case it returns non-zero value instead.